flask_unchained/bundles/oauth/config.py

```diff
@@ -0,0 +1,27 @@
+from flask_unchained import BundleConfig
+
+
+class Config(BundleConfig):
+    """
+    Default configuration settings for the OAuth Bundle.
+    """
+
+    pass
+
+
+class TestConfig(Config):
```

Comment:
While there are no tests I think we can just leave this config out. I found [this old SO post](https://stackoverflow.com/questions/18827985/mocking-oauth-providers-while-testing) on mocking OAuth providers but I haven't tried playing around with it yet...

Reply:
I added the testconfig as an example.
Where would you suggest to document it?

Reply:
right, docs. i think the best place would probably be to start a new sphinx page:

```
# docs/bundles/oauth.rst

OAuth Bundle
------------

... add docs here in restructured text format
```

And add it to the `toctree` in `docs/bundles/index.rst` (ordered alphabetically)

As for building the docs, you might need to install deps: `pip install flask-unchained[docs]`
And then to build/serve them,

```bash
cd your-flask-unchained-root/docs
sphinx-autobuild . _build/html
```
And browse to [http://localhost:8000](http://localhost:8000)